src/roman_datamodels/testing.py

```diff
@@ -109,13 +109,18 @@ def _assert_value_is_copy(value1, value2, deepcopy=False):
         elif isinstance(value1, Model):
             assert value1 is not value2
         else:
-            # Hashable objects cannot be copied as they are immutable.
+            # If not hashable, then can be copied.
             try:
                 hash(value1)
             except TypeError:
                 assert value1 is not value2
             else:
-                assert value1 is value2
+                # Normally hashable objects do not deepcopy.
+                # GWCS violates this.
+                if isinstance(value1, gwcs.WCS) and isinstance(value2, gwcs.WCS):
+                    assert True
```

Comment:
```suggestion
                    assert value1 is not value2
```
Admittedly this is not a fix for the issue you mentioned in https://github.com/spacetelescope/roman_datamodels/pull/477/files#r2027537153 but at least this will catch the case where both wcses are the same object (and not a copy).

Reply:
I'm not seeing this suggestion applied. Was it causing an issue?